### PR TITLE
Update and tidy up the documentation.

### PR DESCRIPTION
**Pull request by reason-bv**
Update the docs:

* Fix line width.
* Fix a broken link.
* Remove `/docs` since it wasn't being used.
* Etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,15 +2,22 @@
...
-**This is a public repository.** Please see the [contribution guidelines]](./CONTRIBUTING.md) for details on contributing to this repo.
+**This is a public repository.** Please see the [contribution guidelines][1] for
+details on contributing to this repo.
 
 ## Installation
 
-You will need npm to add this to your project; it is installed when you install Node, so it is likely that you already have it. If not, you can install Node using an [installer](https://nodejs.org/download/), or by using your favorite package manater (such as Homebrew).
+You will need NPM to add this to your project; it is installed when you install
+Node, so it is likely that you already have it. If not, you can install Node
+using an [installer][2], or by using your favorite package manater (such as
```
Comment (tnunamak) [inline README.md]: s/manater/manager

File: README.md [hunk 2]
```diff
@@ -2,15 +2,22 @@
 
 # bv-ui-core
 
-This project provides a central location for common Bazaarvoice UI code. It is intended to be installed into a project via npm. Individual modules are authored as CommonJS modules, to be consumed by Webpack or another build tool that can ingest CommonJS.
+This project provides a central location for common Bazaarvoice UI code. It is
+intended to be installed into a project via npm. Individual modules are authored
```
Comment (tnunamak) [inline README.md]: s/npm/NPM

Comment (rmurphey) [inline README.md]: @tnunamak can you open a new PR with these changes? This PR has already landed. Thanks :)

Comment (tnunamak) [inline README.md]: Yep :)

**Pull request closed without merging**